run_common.go

```diff
@@ -262,6 +262,7 @@ func (b *Builder) configureUIDGID(g *generate.Generator, mountPoint string, opti
 	}
 	g.SetProcessUID(user.UID)
 	g.SetProcessGID(user.GID)
+	g.AddProcessAdditionalGid(user.GID)
```

Comment:
NIce catch @flouthoc .  I could have sworn I looked for SetProcessGID, but didn't find it....